src/setup.py

```diff
@@ -19,7 +19,7 @@
 
 setup(
     name='katello-agent',
```

Comment:
This means it'll be published as katello-agent on PyPI. Should we rename this to katello-host-tools instead? That probably makes it a major version bump though.